release_notes/ocp-4-18-release-notes.adoc

```diff
@@ -552,6 +552,27 @@ With this release, the Agent-based Installer supports creating assets that can b
 
 For more information, see xref:../installing/installing_with_agent_based_installer/installing-using-iscsi.adoc#installing-using-iscsi[Preparing installation assets for iSCSI booting].
 
+[id="ocp-4.18-postinstallation-configuration_{context}"]
+=== Postinstallation configuration
+
+[id="ocp-4.18-migrate-x86-cp-to-arm-arch_{context}"]
+==== Migrating the x86 control plane to arm64 architecture on {aws-full}
+
+With this release, you can migrate the control plane in your cluster from `x86` to `arm64` architecture on {aws-first}. For more information, see xref:../updating/updating_a_cluster/migrating-to-multi-payload.adoc#migrating-from-x86-to-arm64-cp_updating-clusters-overview[Migrating the x86 control plane to arm64 architecture on Amazon Web Services].
+
+[id="ocp-4.18-support-for-config-image-stream-import-beh_{context}"]
+==== Support for configuring the image stream import mode behavior (Technology Preview)
+
+This feature provides a new field `imageStreamImportMode` in the `image.config.openshift.io/cluster` resource. The `imageStreamImportMode` field controls the import mode behavior of image streams. You can set the `imageStreamImportMode` field to either of the following values:
```

Comment:
I thought this was already GA? Maybe we mean imageStreamImporMode defaults to PreserveOriginal with clusters using the multi-payload? @Prashanth684

Reply:
This field is brand new and behind a featuregate. The importmode in `Imagestreams` is what was implemented during the multi-arch compute project. This is a knob in the configuration that controls the setting of that mode globally.